### PR TITLE
Fix #1221 - Split crypto code into its own module

### LINKED ISSUE #1221: Split Schnorr code into its own repository
The Schnorr code is pretty simple and self contained, and could be extracted in its own repository.
This could allow us to get reviews on it as well.

### PR DESCRIPTION
**Pull request by Geod24**
Uses my branch just to test the CI and soundness. Targets https://github.com/bpfkorea/crypto/pull/1

### CODE REVIEW

**Comment by Geod24** [comment]
Using v0.1.0 now, should be ready.

File: source/agora/test/Crypto.d [hunk 1]
```diff
@@ -0,0 +1,57 @@
+/*******************************************************************************
+
+    Contains extra tests for the `crypto` library
+
+    Those tests do not belong to the crypto library, but ensures that it
+    integrates correcctly with utilities we use.
```
Comment (kchulj) [inline source/agora/test/Crypto.d]:
```suggestion
    integrates correctly with utilities we use.
```

Comment (Geod24) [reply source/agora/test/Crypto.d]: Fixed

**Comment by Geod24** [comment]
Green

**Review by kchulj — APPROVED** [review]
Nice TY.

**Pull request merged**